src/core/software/pupil-capture.md

```diff
@@ -116,30 +119,31 @@ Make sure that the users pupil is properly detected and tracked. Make sure that
     </div>
 </div>
 
-### Calibration Methods
-Before starting calibration, ensure that eye(s) are robustly detected and that the headset is comfortable for the participant.
+### Choreographies
 
-#### Screen Marker Calibration
-This is the default method, and a quick way to get started. It is best suited for close range eye-tracking in a narrow field of view.
+All calibrations work in the way that specific markers are presented to the participant while the participant is instructed to look at the markers as closely as possible. The way in which markers are presented is called `Choreography`. Pupil offers different choreographies for common use cases.
+
+#### Screen Marker Calibration Choreography
+This is the default choreography, and a quick way to get started. It is best suited for close range eye-tracking in a narrow field of view.
 
 <video width="100%" controls class="mb-5">
   <source src="../../media/core/videos/clb-s.mp4" type="video/mp4">
 </video>
 
-1. Select `Screen Marker Calibration`
-1. Select your `Monitor` (if more than 1 monitor)
-1. Toggle `Use fullscreen` to use the entire extents of your monitor (recommended). You can adjust the scale of the pattern for a larger or smaller calibration target.
-1. Press `c` on your keyboard or click the blue circular `C` button in the left hand side of the world window to start calibration.
-1. Follow the marker on the screen with your eyes. Try to keep your head still during calibration.
-1. The calibration window will close when calibration is complete.
+1. Select the `Screen Marker` choreography
+2. Select your `Monitor` (if more than 1 monitor)
+3. Toggle `Use fullscreen` to use the entire extents of your monitor (recommended). You can adjust the scale of the pattern for a larger or smaller calibration target.
```

Comment:
Is `Use fullscreen` the default?

Reply:
It is.
Maybe we remove this point and add it below?
See suggestions there.